packages/cli/src/commands/pg/settings/auto-explain.ts

```diff
@@ -0,0 +1,36 @@
+import {Command, flags} from '@heroku-cli/command'
+import {Args} from '@oclif/core'
+import heredoc from 'tsheredoc'
+import {generate, boolean} from '../../../lib/pg/setter'
+// ref: https://www.postgresql.org/docs/current/auto-explain.html
+function explain(setting: {value: string}) {
+  if (setting.value) {
+    return 'Execution plans of queries will be logged for future connections.'
+  }
+
+  return 'Execution plans of queries will not be logged for future connections.'
+}
+
+export default class AutoExplain extends Command {
+  static topic = 'pg';
+  static description = heredoc(`
+  Automatically log execution plans of queries without running EXPLAIN by hand.
+  The auto_explain module is loaded at session-time so existing connections will not be logged.
+  Restart your Heroku app and/or restart existing connections for logging to start taking place.'
+  `)
+
+  static flags = {
+    app: flags.app({required: true}),
+    remote: flags.remote(),
+  }
+
+  static args = {
+    database: Args.string(),
+    value: Args.string(),
```

Comment:
Instead of doing some interpretation here with a custom boolean function, should we just declare this argument as `boolean`? As in:
```javascript
value: Args.boolean()
```

That would conform with other cli commands.

Reply:
If it is ok to change the original functionality, I'd def like to clean this up. There are a number of weird things happening with these commands but I was hesitant to make significant changes since @k80bowman prefers to migrate as-is, even if defects exist.

Reply:
Yeah I'm ok either way I guess. By default, I'm in agreement with the "no refactors, no changes in behavior" mantra for this project. But I don't think we should be absolutist about it. If we can deliver value with minimal risk/effort, why not? In this situation, I think it's worthwhile to create some conformity with how other CLI commands work. We're not really changing the API of the command (although we are changing the order of the args as discussed in another thread), though we are restricting the inputs to be of boolean (that is, true, or false). We would not accept "on" or "off" or whatever, but I also don't think it's a good thing overall for the CLI for some commands to accept certain boolean-ish inputs while others don't.

Anyway, I'll step off my soapbox now. If we want to continue with the current implementation, I'll approve.

Reply:
+1